core/stylesheets/custom-properties.tid

```diff
@@ -0,0 +1,30 @@
+title: $:/core/stylesheets/custom-properties
+tags: [[$:/tags/Stylesheet]]
+list-before: 
+
+\rules only transcludeinline macrocallinline html transcludeblock
+
+/* Tiddlywiki's CSS properties */
+
+:root {
+	<$list filter="[[$:/palettes/Vanilla]indexes[]]">
+	--tpc-<<currentTiddler>>: <$transclude $variable="colour" $mode="inline" name=<<currentTiddler>>/>;
+	</$list>
```

Comment:
I think the rendered output created with the following code looks a bit better, since it avoids the extra newlines.

```
:root {
<$list filter="[[$:/palettes/Vanilla]indexes[]]">
	--tpc-<<currentTiddler>>: <$transclude $variable="colour" $mode="inline" name=<<currentTiddler>>/>;</$list>
```